### PR TITLE
Release Notes CI 1.56

### PR DESCRIPTION
**Pull request by nofarblue**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: Release Notes CI 1.56
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/continuous-integration.md [hunk 1]
```diff
@@ -42,6 +42,29 @@ If you have any questions or need assistance with the whitelisting process, plea
...
+- Fixed an issue where the DRONE_REPO_OWNER built-in environment variable pointed to the wrong owner when the CI pipeline was triggered by a tag event. A new feature flag (CI_DRONE_REPO_OWNER) has been introduced to ensure DRONE_REPO_OWNER is correctly extracted from the repository URL (CI-14468).
+- The volume size field, in CI stage Kubernetes infrastructure settings, has been updated to support expressions, improving user experience and functionality (CI-14043, ZD-69169).
+- Updated Alpine image version to address security vulnerabilities in images plugins/gcs:1.6.0 and plugins/artifact-metadata-publisher:2.0 (CI-14897, ZD-71880).
+- Corrected the artifact URL output in the "Build and Push to GAR" step to ensure the published image URL is formatted correctly (CI-14917, ZD-71930).
+- Added support for expressions in the memory and CPU fields under the codebase configuration, enabling dynamic configuration of resource limits (CI-15043).
+- Resolved an issue where Cache Intelligence in self-hosted builds stopped functioning as expected. The fix involved adding source paths to restore steps, ensuring the feature now works properly (CI-15201, ZD-73305).
```
Comment (hemanthmantri) [inline release-notes/continuous-integration.md:56]: Cache intel in self-hosted builds when `paths` field is specified.

**Review by hemanthmantri — APPROVED** [review]
Minor nit. LG otherwise.

**Comment by dewan-ahmed** [comment]
Screenshot 2024-12-06 at 10 28 22 AM

Preview looks good.

**Review by dewan-ahmed — APPROVED** [review]
LGTM

**Pull request merged**